xarray/util/generate_aggregations.py

```diff
@@ -237,15 +235,15 @@ def __init__(
         if bool_reduce:
             self.array_method = f"array_{name}"
             self.np_example_array = """
-        ...     np.array([True, True, True, True, True, False], dtype=bool),"""
+        ...     np.array([True, True, True, True, True, False], dtype=bool)"""
 
         else:
             self.array_method = name
             self.np_example_array = """
-        ...     np.array([1, 2, 3, 1, 2, np.nan]),"""
+        ...     np.array([1, 2, 3, 1, 2, np.nan])"""
```

Comment:
Do you know why this is needed? Why wasn't it erroring earlier? Or is pre-commit fixing it locally?

Reply:
I believe I made this change initially to make pre-commit happy. Not sure it's needed anymore since the configuration changed.

Reply:
I see the comma is added below in the template so it makes sense now. Sorry for the noise!